server/db/models/index.js

```diff
@@ -1,18 +1,41 @@
 const User = require('./user')
+const Pothole = require('./pothole')
+const Order = require('./order')
+const Comment = require('./comment')
+const Crew = require('./crew')
+const Upvote = require('./upvote')
+
+Pothole.belongsTo(User, {foreignKey: 'reporterId'})
```

Comment:
Can we add the equivalent hasMany for a user so that they can access the magic methods?

Reply:
👍